### PR TITLE
I added a few line of code

### PR DESCRIPTION
**Pull request by joshuaudeh**

### CODE REVIEW

**Comment by whirmill** [comment]
I said that there's a conflict,how do we do?

File: src/App.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
```
Comment (whirmill) [inline src/App.js:19]: leave this line as it is

File: src/components/HelloWorld.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ function HelloWorld(props) {
     <div>
       <h1>Hello world</h1>
       <h2>this is a test</h2>
+      <h1>Hello world!!!</h1>
```
Comment (joshuaudeh) [inline src/components/HelloWorld.js:8]: please leave as it is

Comment (joshuaudeh) [reply src/components/HelloWorld.js:8]: it is very important

Comment (whirmill) [reply src/components/HelloWorld.js:8]: ok, i'll leave it as it is

**Pull request merged**